server/src/main/java/io/druid/server/initialization/jetty/JettyServerModule.java

```diff
@@ -253,6 +255,31 @@ public void start() throws Exception
           @Override
           public void stop()
           {
+            long startTime = System.currentTimeMillis();
+            long gracefulShutDownTimeout = (long) server.getAttribute(GRACEFUL_SHUTDOWN_TIMEOUT);
+
+            while (activeConnections.get() > 0 &&
```

Comment:
I think activeConnections is number of socket connections setup and not necessarily number of requests in progress or queued. connection pooling clients would generally never close connections and the count here might not reduce.
is there a more reliable way of looking at number of requests being processed instead ?